plugins/tracker-resources/src/components/issues/KanbanView.svelte

```diff
@@ -0,0 +1,182 @@
+<!--
+// Copyright © 2022 Hardcore Engineering Inc.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import contact from '@anticrm/contact'
+  import { Class, Doc, FindOptions, Ref, WithLookup } from '@anticrm/core'
+  import { Kanban } from '@anticrm/kanban'
+  import { getClient } from '@anticrm/presentation'
+  import { Issue, IssuesGrouping, Team, ViewOptions } from '@anticrm/tracker'
+  import { Button, Icon, IconAdd, showPopup, Tooltip } from '@anticrm/ui'
+  import { focusStore, ListSelectionProvider, SelectDirection, selectionStore } from '@anticrm/view-resources'
+  import ActionContext from '@anticrm/view-resources/src/components/ActionContext.svelte'
+  import Menu from '@anticrm/view-resources/src/components/Menu.svelte'
+  import { onMount } from 'svelte'
+  import tracker from '../../plugin'
+  import { getKanbanStatuses } from '../../utils'
+  import CreateIssue from '../CreateIssue.svelte'
+  import AssigneePresenter from './AssigneePresenter.svelte'
+  import IssuePresenter from './IssuePresenter.svelte'
+  import PriorityEditor from './PriorityEditor.svelte'
+
+  export let currentSpace: Ref<Team>
+  export let baseMenuClass: Ref<Class<Doc>> | undefined = undefined
+  export let viewOptions: ViewOptions
+  export let query = {}
+
+  $: ({ groupBy, shouldShowEmptyGroups, shouldShowSubIssues } = viewOptions)
+  $: resultQuery = {
+    ...(shouldShowSubIssues ? {} : { attachedTo: tracker.ids.NoParent }),
+    space: currentSpace,
+    ...query
+  }
+
+  const client = getClient()
+
+  function toIssue (object: any): WithLookup<Issue> {
+    return object as WithLookup<Issue>
+  }
+
+  const options: FindOptions<Issue> = {
+    lookup: {
+      assignee: contact.class.Employee
```

Comment:
IssuePresenter now require space lookup to be present, please check if it working with new codebase.

Reply:
Added